db/migrate/20200211154433_remove_end_user_plans.rb

```diff
@@ -0,0 +1,14 @@
+class RemoveEndUserPlans < ActiveRecord::Migration
+  def change
+    remove_column :cinstances,  :end_user_required, :boolean
```

Comment:
I think table & column are sufficient for migration that does removal @josemigallas as in `remove_column :table_name, :column_name`

Reply:
We should not do that first, we should remove programmatically